test/sanity/code-smell/no-assert.py

```diff
@@ -19,14 +19,18 @@ def main():
         if path in skip:
             continue
 
-        with open(path, 'r') as f:
+        with open(path, 'rb') as f:
             for i, line in enumerate(f.readlines()):
                 matches = ASSERT_RE.findall(line)
 
                 if matches:
                     lineno = i + 1
-                    colno = line.index('assert') + 1
-                    print('%s:%d:%d: raise AssertionError instead of: %s' % (path, lineno, colno, matches[0][colno - 1:]))
+                    colno = line.index(b'assert') + 1
```

Comment:
The b_ convention was missed for line.

Reply:
I'm conflicted as to whether all of the matches style variables should also have a b_... I kinda lean towards yes.  They aren't byte strings but a container of byte strings but the convention is designed to make it obvious when you're combining text and bytes.  That doesn't work for matches unless you've marked that it contains byte strings.